commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java

```diff
@@ -274,6 +334,41 @@ else if (status ==
 		}
 	}
 
+	protected CommerceAddress updateCommerceShipmentAddress(
+			CommerceShipment commerceShipment, String name, String description,
+			String street1, String street2, String street3, String city,
+			String zip, long commerceRegionId, long commerceCountryId,
+			String phoneNumber)
+		throws PortalException {
+
+		CommerceAddress commerceAddress =
+			commerceShipment.fetchCommerceAddress();
+
+		if ((commerceAddress != null) &&
+			name.equals(commerceAddress.getName()) &&
```

Comment:
Couldn't these variables be passed in as `null`? We check for nullness in `commerceAddressLocalService.addCommerceAddress`, so it seems like we shouldn't break here if these values are `null`. We could just sidestep the issue with `Objects.equals`.